web/src/main/java/com/epam/rft/atsy/web/configuration/WebConfiguration.java

```diff
@@ -9,23 +10,24 @@
 import org.springframework.context.support.ReloadableResourceBundleMessageSource;
 import org.springframework.http.converter.HttpMessageConverter;
 import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.web.servlet.HandlerExceptionResolver;
 import org.springframework.web.servlet.LocaleResolver;
 import org.springframework.web.servlet.ViewResolver;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurationSupport;
 import org.springframework.web.servlet.i18n.CookieLocaleResolver;
 import org.springframework.web.servlet.i18n.LocaleChangeInterceptor;
 import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.json.MappingJackson2JsonView;
 
+import java.util.List;
 import java.util.Locale;
 
 @Configuration
-@EnableWebMvc
 @ComponentScan("com.epam.rft.atsy.web")
 @Import({ServiceConfiguration.class})
-public class WebConfiguration extends WebMvcConfigurerAdapter {
+public class WebConfiguration extends WebMvcConfigurationSupport {
```

Comment:
By extending WebMvcConfigurationSupport it can be two WebMvcConfigurationSupport instances registered,  one by @EnableWebMvc and this one.

As the default behaviour is to detect the WebMvcConfigurerAdapters and use them to configure the SpringMVC, I think it is more safe to use the WebMvcConfigurerAdapter.

Also I can see that you would like to reuse the addDefaultHandlerExceptionResolvers method, however I think you should define the default handler exception resolvers here again.